module/awesomebooks.js

```diff
@@ -0,0 +1,59 @@
+// check if the localStorage has a item
+let collections = JSON.parse(localStorage.getItem('books-collection')) || [];
+
+// create collection as page load
+const createCollection = (book, bookContainer) => {
+  const content = `
+    <article class="book">
+      <p>"${book.name.charAt(0).toUpperCase() + book.name.slice(1)}" by ${book.author.charAt(0).toUpperCase() + book.author.slice(1)}</p>
+      <button id="${book.id}">Remove</button>
+    </article>
+  `;
+  // append to the DOM
+  bookContainer.insertAdjacentHTML('beforeend', content);
+};
+
+// add event listener to button
+const removeBook = (id) => {
```

Comment:
- [ ]  [optional] It would have been nice if you added this delete book function as an individual module, this will enable easier reading and scalability of your code.